streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -153,7 +153,7 @@ public void restore() {
             }
         } finally {
             consumer.assign(Collections.<TopicPartition>emptyList());
-            log.debug("{} Took {} ms to restore all active states", logPrefix, time.milliseconds() - start);
+            log.info("{} Completed restore all active states from changelog topics {} in {}ms ", logPrefix, needsRestoring.keySet(), time.milliseconds() - start);
```

Comment:
This is not one entry per state store, but one entry for all stores.

In this patch I tried to make all INFO level logging to be on StreamThread level and only for aggregated events.